src/game/common/globaldata.cpp

```diff
@@ -699,13 +699,6 @@ GlobalData::GlobalData()
     m_hardSoloAIHealthBonus = 1.0f;
```

Comment:
this made me notice the health bonuses were arrays

Reply:
Are they? The INI parse function doesn't look to be using it as an array L319.

Reply:
They are, ill pr the fix later